src/kernel_abi.h

```diff
@@ -1815,6 +1878,20 @@ struct X86Arch : public BaseArch<SupportedArch::x86, WordSize32Defs> {
 
 #include "SyscallEnumsX86.generated"
 
+  // The same as x86_64
+  static const int PTRACE_GETREGS = Arch64::PTRACE_GETREGS;
+  static const int PTRACE_SETREGS = Arch64::PTRACE_SETREGS;
+  static const int PTRACE_GETFPREGS = Arch64::PTRACE_GETFPREGS;
+  static const int PTRACE_SETFPREGS = Arch64::PTRACE_SETFPREGS;
+  static const int PTRACE_GETFPXREGS = Arch64::PTRACE_GETFPXREGS;
+  static const int PTRACE_SETFPXREGS = Arch64::PTRACE_SETFPXREGS;
+  static const int PTRACE_OLDSETOPTIONS = Arch64::PTRACE_OLDSETOPTIONS;
+  static const int PTRACE_GET_THREAD_AREA = Arch64::PTRACE_GET_THREAD_AREA;
+  static const int PTRACE_SET_THREAD_AREA = Arch64::PTRACE_SET_THREAD_AREA;
+  static const int PTRACE_ARCH_PRCTL = Arch64::PTRACE_ARCH_PRCTL;
```

Comment:
You can just delete this and let it fall through to the default negative value, since 32 bit x86 doesn't have PTRACE_ARCH_PRCTL.

Reply:
will do